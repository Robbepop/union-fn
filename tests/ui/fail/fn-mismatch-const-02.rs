fn main() {}

#[union_fn::union_fn]
trait Foo {
    const fn foo() {}
    fn bar() {}
}
