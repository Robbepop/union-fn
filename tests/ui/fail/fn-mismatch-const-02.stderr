error: encountered mismatch in constness for #[union_fn] method
 --> tests/ui/fail/fn-mismatch-const-02.rs:6:5
  |
6 |     fn bar() {}
  |     ^^

error: mismatch with this method
 --> tests/ui/fail/fn-mismatch-const-02.rs:5:5
  |
5 |     const fn foo() {}
  |     ^^^^^
